diff --git a/src/lib/default-project/project-data.js b/src/lib/default-project/project-data.js
index 9ee63b237..7303cee95 100644
--- a/src/lib/default-project/project-data.js
+++ b/src/lib/default-project/project-data.js
@@ -79,8 +79,8 @@ const projectData = translateFunction => {
                         bitmapResolution: 1,
                         md5ext: 'bcf454acf82e4504149f7ffe07081dbc.svg',
                         dataFormat: 'svg',
-                        rotationCenterX: 48,
-                        rotationCenterY: 50
+                        rotationCenterX: 72.5,
+                        rotationCenterY: 27.5
                     },
                     {
                         assetId: '0fb9be3e8397c983338cb71dc84d0b25',
@@ -88,8 +88,8 @@ const projectData = translateFunction => {
                         bitmapResolution: 1,
                         md5ext: '0fb9be3e8397c983338cb71dc84d0b25.svg',
                         dataFormat: 'svg',
-                        rotationCenterX: 46,
-                        rotationCenterY: 53
+                        rotationCenterX: 72.5,
+                        rotationCenterY: 27.5
                     }
                 ],
                 sounds: [
@@ -116,6 +116,7 @@ const projectData = translateFunction => {
         meta: {
             semver: '3.0.0',
             vm: '0.1.0',
+            origin: 'https://xcratch.github.io/editor',
             agent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Safari/537.36' // eslint-disable-line max-len
         }
     });
